### DISCUSSION TITLE
Autowarmth plugin on regular LCD colored screen.

### DISCUSSION THREAD

**Discussion by Zeyadas** [koreader/koreader#8343] [Q&A] [answered]
The new autowarmth plugin #8129 just activate the night mood on LCD screen? I mean no (warmth)?
I'm here with (several Samsung Android devices).

**Comment by pazos** [comment, marked as answer]
Nope. Warmth on e-ink devices is an array of orange leds (or a certain mix of rgb leds).

There's no way to get this plugin working on lcd android devices without https://github.com/koreader/koreader/issues/5768. Once the feature is there (that means: a mostly transparent surface on top of the KOReader window that can *tint* lower surfaces) the plugin will work as expected.

 Patches for this are very welcome. Both options (integrated in the app or calling an external app that does the job) are ok, IMHO

**Reply by Zeyadas** [reply]
As I said in #5768, I use a 3rd party app for that (twilight) or the android reading mode, but when the autowarmth plugin came here I thought It's solved somehow for LCD screens on android, after trying it just activated the night mode, and I had to ask you to confirm it's just night mode not warmth mode for LCD screens. 

Thanks guys.

**Reply by Frenzie** [reply]
@zwim Probably still needs something like https://github.com/koreader/koreader/blob/master/plugins/autosuspend.koplugin/main.lua#L3-L11 at the start.

**Reply by ghost** [reply]
On an Android device with LCD display the plugin is just called auto nightmode, no word about autowarmth (except in the about text. But even there it is stated "if possible").
You can not set a warmth, only nightmode is selectable.

**Reply by ghost** [reply]
@Frenzie I think the plugin is OK so.

If the device has natural lights the plugin can do warmth and nightmode.
If the device doesnt have natural lights it only does nightmode. 

Tested on Tolino and my phone.

**Reply by Frenzie** [reply]
@zwim Right, my bad. I forgot about the phrasing in the original question by the time I posted that. :blush:
